### ISSUE TITLE
Unable to start in server mode (docker)

### ISSUE THREAD

**Issue by wanderleyteixeira** [euzu/m3u-filter#81]
**Describe the bug**
I am spin up a m3u-filter container in a Synology Docker env. Not sure what is missing here. Any guidance would be appreciated.

**Screenshots**
![Screenshot 2024-08-28 at 1 52 45 PM](https://github.com/user-attachments/assets/088409a9-656a-43f9-8236-fd1bf8bae5f8)

**Additional context**
![Screenshot 2024-08-28 at 1 53 05 PM](https://github.com/user-attachments/assets/6d8d294b-4bfc-40b9-8278-055c472ef148)

**Comment by euzu** [comment]
@wanderleyteixeira   you need config files matching your needs and settings. Examples are in the source. YOu can read the details in the wiki.  If you do not provide fully functional config files m3u filter wont work.

if you use docker to start m3u-filter, you need to mount a folder for the config files. You can find an example docker-compose file under the docker directory.
Asuming you have placed the files under `/opt/m3u-filter/` the compose file should look like:

```yaml
version: '3'
services:
  m3u-filter:
    container_name: m3u-filter
    image: ghcr.io/euzu/m3u-filter:latest
    working_dir: /
    volumes:
      - /opt/m3u-filter/data:/data
      - /opt/m3u-filter/config:/config
      - /opt/m3u-filter/backup:/backup
      - /opt/m3u-filter/downloads:/downloads
    environment:
      - TZ=Europe/Paris
    ports:
      - "8901:8901"
    restart: unless-stopped
```

**Comment by wanderleyteixeira** [comment]
Hey, thanks for the quick reply. I do have those settings. I added a config.yml with `api: {host: localhost, port: 8901, web_root: ./web}` but now it asks for source.yml. 

I also tried to change the arg to `./m3u-filter -s -p /config` but still asks for source.yml

![Screenshot 2024-08-28 at 2 37 00 PM](https://github.com/user-attachments/assets/f4fe49b9-bc74-4287-bdbb-ac76291e61c3)
![Screenshot 2024-08-28 at 2 37 32 PM](https://github.com/user-attachments/assets/03911b7b-72e3-426e-a6de-ce1eb99120ff)
![Screenshot 2024-08-28 at 2 37 49 PM](https://github.com/user-attachments/assets/49e755b3-79cd-4edb-b58e-284c8cbebc55)

**Comment by euzu** [comment]
@wanderleyteixeira  you need to place the  config directory under  `docker/m3u-filter`.
You dont need to mount the web folder, it is included in the image.

the config directory should contain the config files.

**Comment by wanderleyteixeira** [comment]
Thanks. Yes, the `config` directory is mounted under `docker/m3u-filter`. Also, it does read the config.yml in config directory. For example, If it remove the config.yml (from config directory) it gives me an error `M3uFilter error: Could not find file config.yml No such file or directory (os error 2)` as expected. I added the config.yml and I get the following error `M3uFilter error: Could not find file source.yml No such file or directory (os error 2)`
I also removed the web mount point to no change.
